app/src/main/res/values/plugins.xml

```diff
@@ -66,13 +66,13 @@
     <string name="indicator_when">When…</string>
     <string name="choice_start_of">Start of</string>
     <string name="choice_end_of">End of</string>
-    <string name="choice_any_event">Any Events</string>
+    <string name="choice_any_event">Any events</string>
```

Comment:
The original intention was to use capital to refer to the mechanism (rather than a plain word). It would be better if that's consistent (or completely ignore this).
```suggestion
    <string name="choice_any_event">Any Events</string>
```

Reply:
Should that maybe be \'Any Events\' to make that clear?